### PR TITLE
Comment out NodeAffinity.RequiredDuringSchedulingRequiredDuringExecution

### PR DESCRIPTION
**Pull request by davidopp**
because it's not yet implemented.

@wojtek-t to review

cc/ @bgrant0607 @kevin-wangzefeng

### CODE REVIEW

**Comment by wojtek-t** [comment]
@davidopp 

hack/verify-gofmt.sh
!!! 'gofmt -s' needs to be run on the following files: 
./plugin/pkg/scheduler/algorithm/predicates/predicates.go

Can you please fix?

**Comment by wojtek-t** [comment]
Once fixed - LGTM

**Comment by kevin-wangzefeng** [comment]
@davidopp Thanks for the fix, there are also tests that uses NodeAffinity.RequiredDuringSchedulingRequiredDuringExecution in [validation_test.go](https://github.com/davidopp/kubernetes/blob/work1/pkg/api/validation/validation_test.go), [predicates_test.go](https://github.com/kubernetes/kubernetes/tree/master/plugin/pkg/scheduler/algorithm/predicates/predicates_test.go#L890), [test/e2e/scheduler_predicates.go](https://github.com/davidopp/kubernetes/tree/master/test/e2e/scheduler_predicates.go#L422), shall we comment out or change them?

**Comment by davidopp** [comment]
Thanks for noticing those, @kevin-wangzefeng . I should have used case-insensitive grep (or remembered that we actually have tests...)

Also ran 'gofmt -s'

@wojtek-t / @kevin-wangzefeng PTAL.

**Comment by wojtek-t** [comment]
@davidopp - seems that there is still one issue:

=== RUN   TestValidatePod
--- FAIL: TestValidatePod (0.00s)
	validation_test.go:1786: expected success: [metadata.annotations: Invalid value: "scheduler.alpha.kubernetes.io/affinity": invalid character '/' looking for beginning of object key string]

in pkg/api/validation

Can you please take a look?

**Comment by davidopp** [comment]
@k8s-bot ok to test

**Comment by davidopp** [comment]
@k8s-bot test this please

**Comment by davidopp** [comment]
@k8s-bot test this issue: #21220

**Comment by davidopp** [comment]
@k8s-bot test this issue: #IGNORE

**Comment by davidopp** [comment]
@k8s-bot unit test this please issue: #IGNORE

**Comment by davidopp** [comment]
@k8s-bot e2e test this issue: #19997

File: pkg/api/deep_copy_generated.go [hunk 1]
```diff
@@ -1306,15 +1306,6 @@ func DeepCopy_api_NodeAddress(in NodeAddress, out *NodeAddress, c *conversion.Cl
...
 func DeepCopy_api_NodeAffinity(in NodeAffinity, out *NodeAffinity, c *conversion.Cloner) error {
-	if in.RequiredDuringSchedulingRequiredDuringExecution != nil {
-		in, out := in.RequiredDuringSchedulingRequiredDuringExecution, &out.RequiredDuringSchedulingRequiredDuringExecution
-		*out = new(NodeSelector)
-		if err := DeepCopy_api_NodeSelector(*in, *out, c); err != nil {
-			return err
-		}
-	} else {
-		out.RequiredDuringSchedulingRequiredDuringExecution = nil
-	}
```
Comment (mikedanese) [inline pkg/api/deep_copy_generated.go]: Did you delete this manually? I'm trying to debug some weirdness I'm noticing with ./hack/update-generated-deep-copies.sh

Comment (davidopp) [inline pkg/api/deep_copy_generated.go]: No I didn't edit any of the generated files manually.

Comment (mikedanese) [inline pkg/api/deep_copy_generated.go]: Thanks, I opened #21338

**Pull request merged**